packages/vite/src/node/utils.ts

```diff
@@ -775,8 +775,8 @@ function gracefulRename(
     ) {
       setTimeout(function () {
         fs.stat(to, function (stater, st) {
-          if (stater && stater.code === 'ENOENT') gracefulRename(from, to, CB)
-          else cb(er)
+          if (stater && stater.code === 'ENOENT') fs.rename(from, to, CB)
```

Comment:
This was a mistake when doing the translation to our code, see:
https://github.com/isaacs/node-graceful-fs/blob/1f19b0b467e4144260b397343cd60f37c5bdcfda/polyfills.js#L109